provisioner/influx-series.js

```diff
@@ -36,6 +36,15 @@ module.exports.spotRequestsSubmitted = new base.stats.Series({
     // Both the bid and price will be the pre-safety factor number
     bid: base.stats.types.Number,
     price: base.stats.types.Number,
+    securityToken: base.stats.types.String,
+  },
+});
+
+// Store when a security token was retreived
+module.exports.securityTokenRetreived = new base.stats.Series({
+  name: 'AwsProvisioner.SecurityTokenRetreived',
+  columns: {
+    securityToken: base.stats.types.String,
```

Comment:
but we need something to correlate specific instances.  I could alternatively create a different slugid that's used just for this purpose, but we need some sort of unique identifier to be able to link a spot request submitted (which we don't know the ID until after it's submitted) to a worker coming online.

I'll implement a second slugid for this purpose.